### PR TITLE
Few changes in LibCSA

### PR DESCRIPTION
**Pull request by memus**
* Scale for countdown text has been changed.
* Change color to red when countdown is close to 0. 
* Fixed no sound at the beginning of the countdown.

### CODE REVIEW

File: libs/LibCSA/LibCSA.lua [hunk 1]
```diff
@@ -115,11 +119,27 @@ local function Load()
 		end
 	end
 
+	ZO_CenterScreenAnnouncementCountdownLine_PlayCountdownLoopAnimation = ZO_CenterScreenAnnouncementCountdownLine.PlayCountdownLoopAnimation
+	ZO_CenterScreenAnnouncementCountdownLine.PlayCountdownLoopAnimation = function(self)
+		self.countdownControl:SetText(self.currentCountdownTimeS)
```
Comment (memus) [inline libs/LibCSA/LibCSA.lua]: In near future i want to implement custom text instead of 0.

Comment (kyoma) [reply libs/LibCSA/LibCSA.lua]: Yea I want to do that too, together with a "Stopped" text.

Comment (memus) [reply libs/LibCSA/LibCSA.lua]:
```
if self.currentCountdownTimeS == 0 then
    self.countdownControl:SetText(self.customText)
else
    self.countdownControl:SetText(self.currentCountdownTimeS)
end
```
at 0 display for example "DODGE" or something but this should be inserted as additional text to StartCountdown(....., endText) or something like that so we can set this for every countdown.

Comment (memus) [reply libs/LibCSA/LibCSA.lua]: Ah you already thinkig about that. I thought you r asking what i want to do here.

**Pull request merged**